test.js

```diff
@@ -199,9 +234,38 @@ describe('capture-exit', function() {
       });
     });
 
+    it('throws an exit code of the first registered handler which rejects', function() {
```

Comment:
i believe this test can be implemented merely by spying on the real `process.exit` and ensuring it is called at the right and expected time.